graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -1,18 +1,35 @@
-import itertools
+import datetime
+import git
```

Comment:
This is a 3rd party library which is probably not going to be installed -- let's use the package version instead?